src/App.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom'; 
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'; 
```

Comment:
This was such a huge deal! It took me almost 45 minutes to realize that I was successfully changing the URL route - but not seeing any component rendering - because I FORGOT A SWITCH 🤦‍♀️